content/articles/building-a-simple-translation-app-using-python-for-beginners/index.md

```diff
@@ -0,0 +1,151 @@
+Want to build your very own language translation app?
+
+Well, in this tutorial, I'll show you how you can build one yourself using Hugging Face Transformers and Gradio!
+
+### Prerequisite
+
+To understand the contents of this tutorial, a reader needs to be familiar with the Python programming language and machine learning. I'm also going to be using [Google Colab](https://colab.research.google.com) notebook. You need to be familiar with that too.
```

Comment:
Do link external resources for Python and machine learning. You could also link a few articles on machine learning from the Section community itself. If you do decide to do so, the format is as follows:
```
[text](engineering-education/boosting-algorithms-python/)
```
Also, I can see that you've used PyTorch. So, do link a few PyTorch resources as well.